rulebook.tex

```diff
@@ -1729,6 +1747,16 @@ \subsection{Task Fulfillment and Scoring}
         \\
         False delivery & Deliver an intermediate product & $0$
         \\
+        1st competitive delivery
+        & Points for the first delivery for a competitive order. The score is
+        given in addition to the points for the regular delivery.
+        &$+ 10$
+        \\
+        2nd competitive delivery
+        & Point deduction for the second delivery for a competitive order. The
+        points are deducted from the delivery points.
```

Comment:
Here would be a good place to state once more that zero is the minimum. This is what teams may look at for scores more frequently.

Reply:
Yes, makes sense, added.